bindings/python/crocoddyl/multibody/costs/impulse-cop-position.cpp

```diff
@@ -0,0 +1,74 @@
+///////////////////////////////////////////////////////////////////////////////
+// BSD 3-Clause License
+//
+// Copyright (C) 2020, University of Duisburg-Essen, University of Edinburgh
```

Comment:
I don't think that `University of Edinburgh` should appear here, except if @cmastalli has contributed to it.

Reply:
We have been discussing this work, and furthermore the code is based on other previous code.

Reply:
It may might be then better to add `adapted from`.